helpers/filterSeats.js

```diff
@@ -60,19 +67,58 @@ module.exports = function(dust) {
       });
     }
 
+    // get the names of the best two sections from the theatre
+    function getBestSections() {
+      var bestSections = [];
+      // add all section names to an array
+      _.forEach(bandColours, function(bands, section) {
+        bestSections.push(section);
+      });
+      // find the best section
+      bestSections = _.last(bestSections); // just Stalls
+      return bestSections;
+    }
+
+    // compare prices
+    function chosenForYou(ticketRate, ticketRateSection) {
+      var bestSections = getBestSections(bandColours);
```

Comment:
I think this is much neater now.  But you will need to update `function getBestSections()` now that you are passing a parameter to it.